frontend/src/constants/UserConstants.ts

```diff
@@ -0,0 +1,3 @@
+const USER_ROLES = ["User", "Admin", "Staff"];
```

Comment:
```suggestion
const USER_ROLES = ["User", "Admin", "Staff"] as const;
```

If it works, the `as const` lets Typescript know that we want to restrict to these specific strings.